source/MRViewer/ImGuiHelpers.cpp

```diff
@@ -1353,4 +1353,38 @@ void Spinner( float radius, float scaling )
     MR::getViewerInstance().incrementForceRedrawFrames();
 }
 
+bool ModalBigTitle( const char* title, float scaling )
+{
+    auto menu = MR::getViewerInstance().getMenuPluginAs<MR::RibbonMenu>();
+    const MR::RibbonFontManager* fontManager = nullptr;
+    if ( menu )
+        fontManager = &menu->getFontManager();
```

Comment:
i think it is better to use static font accessor, to avoid dynamic casts in `getMenuPluginAs<MR::RibbonMenu>()` in each frame